NBXplorer/DBScripts/017.FixDoubleSpendDetection.sql

```diff
@@ -0,0 +1,137 @@
+﻿CREATE OR REPLACE PROCEDURE fetch_matches(in_code text, in_outs public.new_out[], in_ins public.new_in[], INOUT has_match boolean)
```

Comment:
`public.new_out` fails if the db is not in `public` schema. At previous scripts it was only `new_out`

Reply:
oops fixing it

Reply:
https://github.com/dgarage/NBXplorer/commit/d7a9af1d7acaa29fec69854e79b9831bf5b6d28d